recipes/phonopy/COPYING

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2014, Atsushi Togo
```

Comment:
Can we ask upstream to include a license file? It doesn't seem there is one at all and we appear to creating this for them. 😕